lib/resources/packages.rb

```diff
@@ -23,12 +23,17 @@ class Packages < Inspec.resource(1)
     "
 
     def initialize(pattern)
-      @command = packages_command
-
-      return skip_resource "The packages resource is not yet supported on OS #{inspec.os.name}" unless @command
+      os = inspec.os
+      if os.debian?
+        @pkgs = Debs.new(inspec)
+      elsif %w{redhat suse amazon fedora}.include?(os[:family])
```

Comment:
can we use `os.redhat?`? Otherwise this will not work on scientific or oracle linux